Client/Frontend/Widgets/TwoLineImageOverlayCell.swift

```diff
@@ -0,0 +1,157 @@
+        setupLayout()
+        setupNotifications(forObserver: self,
+                           observing: [.DisplayThemeChanged])
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setupLayout() {
+        separatorInset = UIEdgeInsets(top: 0,
+                                      left: UX.ImageSize + 2 * UX.BorderViewMargin,
+                                      bottom: 0,
+                                      right: 0)
+        selectionStyle = .default
+        stackView.addArrangedSubview(titleLabel)
+        stackView.addArrangedSubview(descriptionLabel)
+
+        containerView.addSubview(leftImageView)
+        containerView.addSubview(stackView)
+        containerView.addSubview(leftOverlayImageView)
+
+        contentView.addSubview(containerView)
+        bringSubviewToFront(containerView)
+
+        NSLayoutConstraint.activate([
+            containerView.topAnchor.constraint(equalTo: contentView.topAnchor),
+            containerView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            containerView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor),
+            containerView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor,
+                                                    constant: -16),
+
+            leftImageView.leadingAnchor.constraint(equalTo: containerView.leadingAnchor,
+                                                   constant: 16),
+            leftImageView.centerYAnchor.constraint(equalTo: containerView.centerYAnchor),
+            leftImageView.heightAnchor.constraint(equalToConstant: 28),
+            leftImageView.widthAnchor.constraint(equalToConstant: 28),
+            leftImageView.trailingAnchor.constraint(equalTo: stackView.leadingAnchor,
+                                                   constant: -16),
+
+            leftOverlayImageView.trailingAnchor.constraint(equalTo: leftImageView.trailingAnchor,
+                                                          constant: 8),
+            leftOverlayImageView.bottomAnchor.constraint(equalTo: leftImageView.bottomAnchor,
+                                                        constant: 8),
+            leftOverlayImageView.heightAnchor.constraint(equalToConstant: 22),
+            leftOverlayImageView.widthAnchor.constraint(equalToConstant: 22),
+
+            stackView.topAnchor.constraint(equalTo: containerView.topAnchor,
+                                           constant: 8),
+            stackView.bottomAnchor.constraint(equalTo: containerView.bottomAnchor,
+                                              constant: -8),
+            stackView.trailingAnchor.constraint(equalTo: containerView.trailingAnchor,
+                                                    constant: -8),
+        ])
+
+        selectedBackgroundView = selectedView
+        applyTheme()
+    }
+
+    func configure() {
+
+    }
```

Comment:
I can remove for now, I wanted to set the views as private and use a configure func instead but this cell is used in several places so I stop because it was doubling the amount of work I can create a separate ticket for this work